convertor/csv2format.py

```diff
@@ -0,0 +1,80 @@
+import csv
+import datetime
+import sys
+
+csvs = sys.argv[1].split(',')
+format = sys.argv[2]
+out = sys.argv[3]
+
+def writePYSEC(mas, startIndex):
+    file_name = 'PYSEC-2022-'+ startIndex
+    f = open(out+file_name, 'a+')
+    f.write('id: '+ file_name + '\n')
+    f.write('summary: '+mas[1] + '\n')
+    f.write('details: '+ mas[4] + '\n')
+    f.write('affected:' + '\n')
+    f.write('- package:' + '\n')
+    f.write('    name: '+ mas[2]+ '\n')
+    f.write('    ecosystem: PyPI'+ '\n')
+    f.write('    purl: pkg:pypi/'+ mas[2]+ '\n')
+    f.write('  ranges:'+ '\n')
+    f.write('  - type: GIT'+ '\n')
+    f.write('    repo: '+ mas[3]+ '\n')
+    f.write('references:'+ '\n')
+    f.write('- type: WEB'+ '\n')
+    f.write('  url: '+ mas[3]+ '\n')
+    try:
+        dt = (datetime.datetime.strptime(mas[0], u'%d.%m.%Y %H:%M:%S').isoformat('T'))
+        f.write('modified: ' + str(dt)+'Z'+ '\n')
+        #f.write('published: ' + str(dt)+'Z')
+    except:
+        f.write('modified:' + '\n')
+        #f.write('published:')
+        print("Date format convert error occured")
+
+    f.close()
+
+def writeCVE(mas, startIndex):
+    file_name = 'CVE-2022-'+ startIndex
+    f = open(out+file_name, 'a+')
+    f.write('identifier: '+ file_name + '\n')
+    f.write('title: '+mas[1] + '\n')
+    f.write('description: '+ mas[4] + '\n')
+    try:
+        dt = (datetime.datetime.strptime(mas[0], u'%d.%m.%Y %H:%M:%S').isoformat('T'))
+        f.write('date: ' + str(dt)+'Z'+ '\n')
+        #f.write('pubdate: ' + str(dt)+'Z')
+    except:
+        f.write('date:' + '\n')
+        #f.write('pubdate:')
+        print("Date format convert error occured")
+    f.write('package_slug: ' + 'pypi/'+ mas[2]+ '\n')
+    f.write('urls:\n')
+    f.write('- ')
+    f.write(mas[3]+'\n')
+    f.close()
+
+prefix = 0
+
+for filename in (csvs):
+    flag = False
+    count = 0
+    prefix = prefix + 1
+    print(filename)
+    with open(filename, encoding="utf8") as f:
+        csv_reader = csv.reader(f)
+        for line_no, line in enumerate(csv_reader, 1):
+            count = count + 1
+            if line_no == 1:
+                print('Header:')
+                print(line)  # header
+                print('Data:')
+            else:
+                # исключаем заголовок
+                if(flag == False):
```

Comment:
fuck my brain, assume we have valid civ on the input, if it is not so - fix manually

Reply:
fix manually ?)) ok

Reply:
fixed